internal/host/statemachine.go

```diff
@@ -295,12 +295,28 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		PostTransition:   th.PostRefreshHost(statusInfoPreparingForInstallation),
 	})
 
+	// Unknown validations
+	installationDiskSpeedUnknown := stateswitch.And(stateswitch.Not(If(InstallationDiskSpeedCheckSuccessful)), If(SufficientOrUnknownInstallationDiskSpeed))
```

Comment:
Why there are two IDs?

Reply:
We need 2 different conditions:
1. To be tested before preparing.  In this case the condition will be successful if it has not yet been tested, or it was tested and it was successful
2. To be tested before starting installation.  In this case the condition should have been tested and the result should be successful.